src/component/Options.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { createRef, useEffect, useRef, useState } from 'react';
```

Comment:
![image](https://user-images.githubusercontent.com/1908017/235774311-e56148a2-59d2-4daf-94bb-d7a45235081f.png)

Visually this part seems a bit crowded. Maybe we can do without the icon and the redundant "Domain Regexes" text.